drawing_3.css

```diff
@@ -0,0 +1,31 @@
+.container{
+	height: 100px;
+	width: 400px;
+}
+.red{
+	background-color: #8B0000;
+	float: right;
+	height: 100px;
+	width: 100px;
+}
+.cyan{
+	background-color: #00FFFF;
+	height: 100px;
+	width: 100px;
```

Comment:
Give this a `float:left;`. Here is another part of software engineering philosophy:

> Explicit is better than implicit

You want your intentions to be clear to another programmer, even if you code which implies things implicitly works.